api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -3731,6 +3731,17 @@ private void enablePrograms(Config config) {
     }
 
     private void configureBasicProgramDashboard(String programUuid) {
```

Comment:
All this is doing is saying - do not add any apps or extensions for the Sierra Leone Pregnancy Patient Dashboard, but otherwise continue to do so.